admin-dev/themes/new-theme/public/translations.bundle.js

```diff
@@ -28177,7 +28177,7 @@ var Component = __webpack_require__(3)(
   /* cssModules */
   null
 )
-Component.options.__file = "/home/bbieri/repos/PrestaShop/admin-dev/themes/new-theme/js/app/widgets/ps-button.vue"
+Component.options.__file = "/Users/thomasleviandier/Documents/projects/prestashop/PrestaShop/admin-dev/themes/new-theme/js/app/widgets/ps-button.vue"
```

Comment:
I've never noticed we have commited this kind of information :/

Reply:
internal usage of webpack DOES expose these informations :/
packaging should delete this, we should work on webpack configuration...

Reply:
Yeah, it's an issue I've been aware of for some time, it's related to webpack and (I think) hot module replacement which we need to deactivate for production builds.
I'm working on it.